#! /bin/sh
patch -p1 -f $* < $0
exit $?

From f504f5805be3cc27e19632e15b29eaaaac0e92e8 Mon Sep 17 00:00:00 2001
From: Boris Pek <tehnick-8@yandex.ru>
Date: Sun, 19 Jun 2016 03:15:43 +0300
Subject: [PATCH 8/8] Fix build of the minimal platform plugin for MS Windows

when qtbase is configured with -fontconfig and -system-freetype.

This fix is necessary after 060e0f6628fd185994911307c59f5355acaaf18f.
Used the same approach as in 16864c42d6bc0ee6b3e3fa03123ef5884557ceea.

Change-Id: Idece0dc11d89e38266c95de1769be751c06324ef
(cherry picked from commit 02d2969fafe5088bb1c80eb58b2736df447326cd)

diff --git a/src/plugins/platforms/minimal/minimal.pro b/src/plugins/platforms/minimal/minimal.pro
index bd6f2d8..4b63461 100644
--- a/src/plugins/platforms/minimal/minimal.pro.orig      2016-07-22 15:53:11.365351113 +0200
+++ b/src/plugins/platforms/minimal/minimal.pro   2016-07-22 15:55:21.772137814 +0200
@@ -11,6 +11,7 @@
 OTHER_FILES += minimal.json

 CONFIG += qpa/genericunixfontdatabase
+win32: DEFINES += QT_NO_FONTCONFIG

 PLUGIN_TYPE = platforms
 PLUGIN_CLASS_NAME = QMinimalIntegrationPlugin
